src/CMake/VisItIncludeVars.cmake

```diff
@@ -50,9 +50,12 @@ set(VISIT_PLUGIN_INCLUDES
     ${VISIT_INCLUDE_DIR}/avt/VisWindow/VisWindow
     ${VISIT_INCLUDE_DIR}/visit_vtk/full
     ${VISIT_INCLUDE_DIR}/visit_vtk/lightweight
-    ${VTK_INCLUDE_DIRS} 
 )
 
+if(VTK_VERSION VERSION_LESS_EQUAL "8.1.0")
```

Comment:
I can see your point. I suppose the version check could be VERSION_EQUAL instead.  Might be more important when testing for '9.1.0' because in our root CMakeLists.txt we specify the minimum VTK version supported is 8.1.0. So any version < 8.1.0 will fail first there.
I will change the VERSION_GREATER_EQUAL '9.1.0' test I have, because I don't know if 9.1.x or 9.2 will work.

Reply:
The next PR will have changes to a lot of CMakeLists.txt with a version check.
Which do you think is better:

```
if(VTK_VERSION VERSION_EQUAL "8.1.0")
  do_something_for_vtk_8()
else()
  do_something_for_vtk9()
endif()
```
or

```
if(VTK_VERSION VERSION_GREATER_EQUAL "9.1.0")
    do_something_for_vtk9()
else()
    do_something_for_vtk8()
endif()
```

Reply:
I prefer the first, since the normal case right now is vtk8.